src/main/java/io/github/thebusybiscuit/sensibletoolbox/utils/WordUtils.java

```diff
@@ -0,0 +1,63 @@
+package io.github.thebusybiscuit.sensibletoolbox.utils;
+
+public class WordUtils {
+
+    public static String wrap(String str, int wrapLength) {
+        return wrap(str, wrapLength, null, false);
+    }
+
+    public static String wrap(String str, int wrapLength, String newLineStr, boolean wrapLongWords) {
```

Comment:
Can I assume this has been taken from the commons lang lib? If so not sure what the rules are there but can you put a comment explaining where you got it from and why. See this similar PR from Yb in dough to see what I mean https://github.com/baked-libs/dough/blob/739f119dcb8c417d3393425dd349ebed7cfb9347/dough-common/src/main/java/io/github/bakedlibs/dough/common/Validate.java#L27

Reply:
It is, yes. I figured the best way to fix this one was a direct copy of what apache-commons did. I couldn't have figured this out otherwise, haha :)